content/zh/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md

```diff
@@ -0,0 +1,274 @@
+---
+layout: blog
+title: "Kubernetes 1.17：稳定"
+date: 2019-12-09T13：00：00-08：00
+slug: kubernetes-1-17-release-announcement
```

Comment:
```suggestion
```
@rudy2steiner When localizing a blog article you need to either translate the slug or to remove it.

Reply:
Leaving the slug as is will create 2 articles that have the same date and same slug but different languages. This makes for a conflict where one language overwrites the other (and the site build won't complain, I'm afraid).